README.md

```diff
@@ -1,2 +1,284 @@
+# Kanban Restaurant
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="./porag-logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+  <h3><b>Kanban Restaurant</b></h3>
+  <h5>The **Kanban Restaurant** is a JavaScript capstone project: A fancy web application based on 2 external APIs with 3 user interfaces that shows food items and have the option to like, comment and order those items. The project is implemented following GitFlow using HTML, CSS, modular JS. Webpack is used as bundler and Jest for Unit testing.<h5>
+</div>
+
+
+# 📗 Table of Contents
+
+- [Kanban Restaurant](#kanban-restaurant)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Kanban Restaurant](#-kanban-restaurant)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [To add dependencies and run linter tests](#to-add-dependencies-and-run-linter-tests)
+    - [Deployment](#deployment)
+  - [👥 Author ](#-author-)
+    - [Authors](#authors)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+
+# 📖 Kanban Restaurant<a name="about-project"></a>
+
+This project **Kanban Restaurant** is a project that demonstrates the use of **HTML & CSS and Javascript Moduler development**.
+
+With External **API consumption**, it also demonstrates the proper use of **Async and Await** to send and receive **data**.
+
+Built using **WEBPACK** following **GitFlow** it is  very simple yet powerful project served by **WEBPACK DEV SERVER** . Along with dynamic modification of the DOM with Events including all the coding best practices. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JS</li>
+  </ul>
+
+   <ul>
+    <li>webpack</li>
+  </ul>
+</details>
+  
+  <!-- - **WEBPACK BASICS HIGHLIGHTS**
+   <img src="./WebpackBasicByPorag.png" alt="webpack basics config highlights" width="auto"  height="auto" /> -->
+ 
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Local webpack dev server</li>
+    <li>Optionally can be Deployed Using GitHub Pages</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Use of medium-fidelity wireframe attached below to create a UI.**
+   <img src="./leaderboard_wireframe.png" alt="Leaderboard wireframe" width="auto"  height="auto" />
+- **Developed UI:**
+   <img src="./leaderboard_Design.png" alt="Leaderboard wireframe" width="auto"  height="auto" />
```

Comment:
- [ ] There are some broken images in your README, please fix them to give you a more professional README.

![Screenshot 2023-06-10 at 18 15 13 (2)](https://github.com/porag-m06/Kanban-Restaurant/assets/64774936/38da356b-4799-4e37-8dd8-5b1fe6868cd5)

Reply:
Sure, will be removing that. Thank you